app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_comments_counter_for_post
+    return unless post.present?
+
+    post.update(comments_counter: post.comments_counter.to_i + 1)
+  end
```

Comment:
- [ ] I will encourage you to add update_comments_counter_for_post to after_save so that it would always be triggered after every save. This is a callback method in Ruby on Rails that gets triggered after an object is saved to the database. In this case, it means the update_comments_counter method will be called automatically after the model is saved.